app/views/missing/report.php

```diff
@@ -136,7 +136,7 @@
                                         <label for="lname">Last name </label>
                                         <input type="text" class="form-control" placeholder="Enter Last Name" id="lname" name="person[lname]" required /><br>
                                         <label for="dob">Date of birth </label>
-                                        <input type="date" placeholder="Enter your DOB" class="form-control" id="dob" type="date" name="person[dob]" /><br>
+                                        <input type="date" placeholder="Enter your DOB (mm/dd/yyyy)" class="form-control" id="dob" type="date" name="person[dob]" pattern="\d{1,2}/\d{1,2}/\d{4}" /><br>
```

Comment:
The type date is html5 standard, which shows date picker in Chrome at least. Firefox doesn't support it yet. 

But, yes. It would be nice if we use date picker here for backward compatibility.